### PR TITLE
Add button on home page to change background color

### PR DESCRIPTION
**Pull request by antoniolinhart**
- Add button on homepage
- Create javascript functions to:
  - Generate a background color
  - Set that color as the HTML page's background
- Remove unnecessary code from script.js

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -50,6 +50,7 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
           </p>
         </div>
       </div>
+      <button type="button" class="btn-primary btn-lg" onclick="setRandomBackgroundColor()">Change the background</button>
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/index.html:53]: I'd recommend keeping Javascript within script.js: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Try using [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) to add an onclick listener to this button.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,22 @@
...
- * Adds a random greeting to the page.
+ * Generates a random color.
+ * @returns {string} A color in HSL (hue, saturation, lightness) format
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function generateRandomColor() {
+  const randomHue = Math.random() * 360; // max value 360
+  const saturation =  (Math.random() * 40) + 40 // max value 100
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:21]: So you want a random value of saturation in the range [40, 80] and a random value of lightness in the range [60, 90]? Also, just wanted to point out that these values will be floating point numbers in the above ranges - I suppose hsl can accept floating point values for its parameters?

Comment (antoniolinhart) [reply portfolio/src/main/webapp/script.js:21]: Yes, they can take in floating point values! I tried adding floor to each of the lines but it does not have any different behavior.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (antoniolinhart) [reply portfolio/src/main/webapp/index.html]: Added! Struggled a bit with figuring out the correct code - didn't know I needed to use an anonymous function

**Pull request merged**